Source/WebCore/rendering/style/RenderStyleSetters.h

```diff
@@ -285,6 +285,7 @@ inline void RenderStyle::setSpecifiedZIndex(int value) { SET_NESTED_PAIR(m_nonIn
 inline void RenderStyle::setStrokeColor(const StyleColor& color) { SET(m_rareInheritedData, strokeColor, color); }
 inline void RenderStyle::setStrokeMiterLimit(float value) { SET(m_rareInheritedData, miterLimit, value); }
 inline void RenderStyle::setStrokeWidth(Length&& width) { SET(m_rareInheritedData, strokeWidth, WTFMove(width)); }
+inline void RenderStyle::setStyleTypeIdentifier(const AtomString& identifier) { SET_NESTED(m_nonInheritedData, rareData, styleTypeIdentifier, identifier); }
```

Comment:
It would be good to assert this does not get set except for right pseudo-elements.

Reply:
will do